.env.test.local

```diff
@@ -1 +1,3 @@
-NEXT_PUBLIC_URL_IO=https://io.italia.it
\ No newline at end of file
+NEXT_PUBLIC_URL_IO=https://io.italia.it
+# MOCK API FOR TEST ENVIRONMENT
+NEXT_PUBLIC_API_MOCK_WEB_PROFILE=true
```

Comment:
As above

Reply:
seems to be something related to something else, sorry, let's remove it